src/YieldDisburser.sol

```diff
@@ -35,7 +35,6 @@ contract YieldDisburser is OwnableUpgradeable {
      *          Public Functions         *
      *
      */
-
```

Comment:
why removing line breaks? it's inconsistent with other comment blocks now

Reply:
Mistake , maybe we need to add a linter , do you know how?

Reply:
if you're using VS code, there is a Solidity linter that should be included in the plugin and the vscode settings in this repo should be configured to autoformat on save.

Reply:
Works!  I will amend this